runtime-util/src/main/java/io/camunda/connector/runtime/util/outbound/OutboundConnectorRegistrationHelper.java

```diff
@@ -37,8 +37,31 @@ public class OutboundConnectorRegistrationHelper {
   public static final Pattern CONNECTOR_FUNCTION_PATTERN =
       Pattern.compile("^CONNECTOR_(.*)_FUNCTION$");
 
+  public static Map<String, String> hardwiredEnvironmentVariables;
+
+  public static void addHardwiredEnvionmentVariable(String key, String value) {
+    if (hardwiredEnvironmentVariables == null) {
+      hardwiredEnvironmentVariables = new HashMap<>();
+    }
+    hardwiredEnvironmentVariables.put(key, value);
+  }
+
+  public static void clearHardwiredEnvionmentVariable() {
+    hardwiredEnvironmentVariables = null;
+  }
+
+  public static Map<String, String> getEnvionmentVariables() {
```

Comment:
Just a little type there (needs to be fixed in the code using this as well)
```suggestion
  public static Map<String, String> getEnvironmentVariables() {
```

Reply:
Good catch!